zetaclient/chains/ton/liteapi/client_test.go

```diff
@@ -0,0 +1,18 @@
+package liteapi
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestHashes(t *testing.T) {
```

Comment:
Not required for this PR , but the coverage by this test can be improved

Reply:
https://github.com/zeta-chain/node/pull/2896/commits/3f8f98958819c72911d6a78660c10490c14e72c6